pyuvsim/telescope.py

```diff
@@ -128,6 +128,31 @@ def __init__(self, beam_list=None, uvb_params=None):
         self.uvb_params.update(list_uvb_params)
         self.uvb_params.update(uvb_params)    # Optional parameters for the UVBeam objects
 
+        x_orientation_list = []
+        for beam in beam_list:
+            if self.string_mode:
+                beam_use = self._str_to_obj(beam)
+            else:
+                beam_use = beam
+
+            if hasattr(beam_use, "x_orientation"):
+                x_orientation_list.append(beam_use.x_orientation)
+        if (
+            len(x_orientation_list) == 0
+            or x_orientation_list.count(None) == len(x_orientation_list)
+        ):
+            self.x_orientation = None
+        elif x_orientation_list.count(x_orientation_list[0]) == len(x_orientation_list):
```

Comment:
i get this syntax and it's fine, but I _prefer_ `if any(x != x_orientation_list[0] for x in x_orientation_list)` for semantic meaning.

Reply:
but here I'm testing that they all are equal, so I think I want
`elif all(x == x_orientation_list[0] for x in x_orientation_list)`

Reply:
sure thing.